tests/ui_tools/test_popups.py

```diff
@@ -1077,42 +1077,42 @@ def test_keypress_stream_members(self, key, widget_size):
                 {"date_created": None},
                 "74 [Organization default]",
                 None,
-                12,
+                18,
```

Comment:
The increase in popup height does not seem to match the number of fields you added below? i.e. 12 + 5 = 17.

Reply:
It was because the key name of last row was long . So I reduced the size.